src/customers/account-dashboard-my-wish-list.md

```diff
@@ -7,6 +7,72 @@ A wish list is a convenient way for customers to keep track of products that the
 ![]({% link images/images-ee/account-dashboard-wishlist.png %}){: .zoom}
 _My Wish List_
 
+{:.ee-only}
+## Create a new wish list
+
+In the storefront, Wish List can be created from your account dashboard, product pages, catalog pages and from the shopping cart.
+
+### Method 1: From a customer account
+
+1. In the sidebar of your account dashboard, choose **My Wish List**.
+
+1. In the upper-right corner, click **Create New Wish List**.
+
+1. Enter the Wish List Name.
+
+1. If you want others to be able to see your wish list, select the **Public Wish List** checkbox.
+
+1. When complete, click <span class="btn">Save</span>.
+
+    ![]({% link images/images-ee/account-dashboard-wishlist-create-new.png %}){: .zoom}
+    _Create New Wish List_{:.ee-only}
```

Comment:
Whole section is `ee` so not needed here

```suggestion
    _Create New Wish List_
```

Reply:
Done ✅